pkgs/top-level/all-packages.nix

```diff
@@ -20960,10 +20960,6 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Cocoa OpenGL IOKit;
   };
 
-  irrlichtmt = callPackage ../development/libraries/irrlichtmt {
-    inherit  (darwin.apple_sdk.frameworks) Cocoa Kernel;
-  };
-
```

Comment:
Usually we add an entry to aliases.nix explaining why the package was removed. In this case I think it's fine either way since afaik irrlichtmt is only used by minetest and its forks.

Reply:
No other package is using irrlichtmt, I checked. Even Voxelands is using upstream irrlicht.

Reply:
Theoretically there could be out‐of‐tree users, but yeah I doubt it’s worth blocking on.

Reply:
Okay, there is [an out‐of‐tree user](https://github.com/lizelive/minetest-nix/blob/11dc2be18e6ded7b7ec9ce6a4d0f3d414587e46c/package/minetest.nix), although it might be broken for other reasons.

If it wouldn’t be too much trouble to just add a `throw "irrlichtmt was removed because blah blah blah";`, it’d be best practice to do so.

Reply:
Done